recipe/meta.yaml

```
@@ -52,7 +68,12 @@ about:
   license_file: LICENSE
   summary: Extensible, parallel implementations of t-SNE
   description: |
-    openTSNE is a modular Python implementation of t-Distributed Stochasitc Neighbor Embedding (t-SNE), a popular dimensionality-reduction algorithm for visualizing high-dimensional data sets.
+    openTSNE is a modular Python implementation of t-Distributed Stochasitc Neighbor Embedding (t
+    -SNE), a popular dimensionality-reduction algorithm for visualizing high-dimensional data sets.
```

Breaking the line on the hyphen in `t-SNE` looks really unfortunate.  Could we wiggle it so `t-SNE` is on one line or the other?

eg.

```suggestion
    openTSNE is a modular Python implementation of t-Distributed Stochasitc Neighbor Embedding
    (t-SNE), a popular dimensionality-reduction algorithm for visualizing high-dimensional data sets.
```